_pages/07-publications.md

```diff
@@ -385,7 +385,12 @@ M. Dischinger, A. Mislove, A. Haeberlen, K. P. Gummadi
 ## 2017
 {:.no_toc}
 
-### [4/12/2017 - TRAI (India)]({{ site.baseurl }}/publications/regulator-filings/ntia/m-lab_ntia_10-11-2016.pdf){:target="_blank"}: TRAI Consultation Paper on Net Neutrality
+### [4/25/2017 - TRAI (India)](http://trai.gov.in/sites/default/files/Measurement_Lab_27_04_2017.pdf){:target="_blank"}: Counter Comments on TRAI Consultation Paper on Net Neutrality
```

Comment:
There's no need to link out to TRAI. We might as well have all our links be M-Lab.